AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/MutationSync/OutgoingMutationQueue/ProcessMutationErrorFromCloudOperation.swift

```diff
@@ -300,7 +300,13 @@ class ProcessMutationErrorFromCloudOperation: AsynchronousOperation {
             return
         }
 
-        storageAdapter.delete(untypedModelType: modelType, modelSchema: modelType.schema, withId: id) { response in
+        guard let modelSchema = ModelRegistry.modelSchema(from: modelName) else {
+            let error = DataStoreError.unknown("Invalid Model \(modelName)", "")
```

Comment:
Recovery suggestion?

Reply:
Changed to invalidModelName error, it has its own recovery suggesiton.